libs/widget-lib/src/cowSwapWidget.ts

```diff
@@ -65,19 +65,22 @@ export function createCowSwapWidget(container: HTMLElement, props: CowSwapWidget
   // 4. Handle widget height changes
   windowListeners.push(...listenToHeightChanges(iframe, params.height))
 
-  // 5. Handle and forward widget events to the listeners
+  // 5. Intercept deeplinks navigation in the iframe
+  windowListeners.push(interceptDeepLinks())
+
+  // 6. Handle and forward widget events to the listeners
   const iFrameCowEventEmitter = new IframeCowEventEmitter(window, listeners)
 
-  // 6. Wire up the iframeRpcProviderBridge with the provider (so RPC calls flow back and forth)
+  // 7. Wire up the iframeRpcProviderBridge with the provider (so RPC calls flow back and forth)
```

Comment:
It was me probably who starting adding numbers here :) 
probably now that there's so may steps, don't make sense any more to keep updating them, so maybe is better just to ditch them?

Reply:
I was thinking about that, but decided to keep them